code/modules/spells/spell_types/mind_transfer.dm

```diff
@@ -9,8 +9,7 @@
 	invocation_type = "whisper"
 	range = 1
 	cooldown_min = 200 //100 deciseconds reduction per rank
-	var/list/protected_roles = list("Wizard","Changeling","Cultist") //which roles are immune to the spell
-	var/paralysis_amount_caster = 20 //how much the caster is paralysed for after the spell
+	var/paralysis_amount_caster = 15 //how much the caster is paralysed for after the spell
 	var/paralysis_amount_victim = 20 //how much the victim is paralysed for after the spell

```

Comment:
Eh, paranoia is pretty much the whole fun about mindswap. Having clearer tells makes it less paranoiaful.

Of course, what we could do is give the wizard a 0 point spell called "Fake Mindswap" which does no swapping, but knocks them both out. You know... for the memes.